circle.yml

```diff
@@ -1,11 +1,23 @@
 machine:
   services:
     - redis
-    - elasticsearch
```

Comment:
CircleCI gave me a ton of trouble. One of the issues was that the default version of Elasticsearch on CircleCI is 1.7.something. Upgrading the gems required upgrading Elasticsearch (not sure why). I also had some trouble with Postgres, which I solved by changing our base image in CI from Ubuntu 12.04 to 14.04.